requirements.txt

```diff
@@ -1,3 +1,3 @@
 flask[async]>=2.2.2
 python-dotenv>=0.20.0
-stytch>=6.0.1
+stytch>=7.6.0
```

Comment:
Use `stytch~=7.6` to ensure it will never go above the MAJOR version

Reply:
Yea I was wondering if the >= was intentional or unintentional. I'll change to the `~=`